set.mm

```diff
@@ -218777,166 +218954,19 @@ net proof size (existence part)? $)
-      BDDXGVSWPCDXGVOWLVRWOXGVIVIVMWKVNGXCXDUNZXGVIUOXGVLEVJVKXCXDUPZUQURXGVPWM
-      VQWNVLEXIXGVNGVIVJXHUQXGVNGVIVKXHUQURUTVAVBVCVHVDVEABCKUGLMVFVG $.
-
-    $d X x y z $.  $d Y x y z $.  $d Z x y z $.
-
-    $( In a distributive lattice, meets distribute over joins.  (Contributed by
-       Stefan O'Rear, 30-Jan-2015.) $)
-    dlatmjdi $p |- ( ( K e. DLat /\ ( X e. B /\ Y e. B /\ Z e. B ) ) ->
-        ( X ./\ ( Y .\/ Z ) ) = ( ( X ./\ Y ) .\/ ( X ./\ Z ) ) ) $=
-      ( vx vy vz wcel cv co wceq wral oveq1 eqeq12d clat isdlat simprbi oveq12d
-      cdlat w3a oveq2d oveq2 oveq1d rspc3v mpan9 ) CUENZKOZLOZMOZBPZDPZUMUNDPZU
-      MUODPZBPZQZMARLARKARZEANFANGANUFEFGBPZDPZEFDPZEGDPZBPZQZULCUANVBKLMABCDHI
-      JUBUCVAVHEUPDPZEUNDPZEUODPZBPZQEFUOBPZDPZVEVKBPZQKLMEFGAAAUMEQZUQVIUTVLUM
-      EUPDSVPURVJUSVKBUMEUNDSUMEUODSUDTUNFQZVIVNVLVOVQUPVMEDUNFUOBSUGVQVJVEVKBU
-      NFEDUHUITUOGQZVNVDVOVGVRVMVCEDUOGFBUHUGVRVKVFVEBUOGEDUHUGTUJUK $.
-  $}
-
-  ${
-    $d K x y z $.
-    $( A distributive lattice is a lattice.  (Contributed by Stefan O'Rear,
-       30-Jan-2015.) $)
-    dlatl $p |- ( K e. DLat -> K e. Lat ) $=
-      ( vx vy vz cdlat wcel clat cjn cfv cmee wceq cbs wral eqid isdlat simplbi
-      cv co ) AEFAGFBQZCQZDQZAHIZRAJIZRSTUCRSUAUCRUBRKDALIZMCUDMBUDMBCDUDUBAUCU
-      DNUBNUCNOP $.
-  $}
-
-  ${
-    $d K x y z $.  $d D x y z $.  $d V x y z $.
-    odudlat.d $e |- D = ( ODual ` K ) $.
-    $( The dual of a distributive lattice is a distributive lattice and
-       conversely.  (Contributed by Stefan O'Rear, 30-Jan-2015.) $)
-    odudlatb $p |- ( K e. V -> ( K e. DLat <-> D e. DLat ) ) $=
-      ( vx vy vz wcel clat cv cjn cfv co cmee wceq wral wa cdlat eqid isdlat
-      cbs latdisd bicomd pm5.32i odulatb anbi1d syl5bb odujoin odumeet 3bitr4g
-      odubas ) BCHZBIHZEJZFJZGJZBKLZMBNLZMUNUOURMUNUPURMUQMOGBUALZPFUSPEUSPZQZA
-      IHZUNUOUPURMUQMUNUOUQMUNUPUQMURMOGUSPFUSPEUSPZQZBRHARHVAUMVCQULVDUMUTVCUM
-      VCUTEFGUSUQBURUSSZUQSZURSZUBUCUDULUMVBVCABCDUEUFUGEFGUSUQBURVEVFVGTEFGUSU
-      RAUQUSABDVEUKAURBDVGUHAUQBDVFUITUJ $.
-  $}
-
-  ${
-    dlatjmdi.b $e |- B = ( Base ` K ) $.
-    dlatjmdi.j $e |- .\/ = ( join ` K ) $.
-    dlatjmdi.m $e |- ./\ = ( meet ` K ) $.
-    $( In a distributive lattice, joins distribute over meets.  (Contributed by
-       Stefan O'Rear, 30-Jan-2015.) $)
-    dlatjmdi $p |- ( ( K e. DLat /\ ( X e. B /\ Y e. B /\ Z e. B ) ) ->
-        ( X .\/ ( Y ./\ Z ) ) = ( ( X .\/ Y ) ./\ ( X .\/ Z ) ) ) $=
-      ( cdlat wcel codu cfv w3a co wceq eqid odudlatb ibi odujoin odumeet sylan
-      odubas dlatmjdi ) CKLZCMNZKLZEALFALGALOEFGDPBPEFBPEGBPDPQUFUHUGCKUGRZSTAD
-      UGBEFGAUGCUIHUDUGDCUIJUAUGBCUIIUBUEUC $.
-  $}
-
 
 $(
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
   Posets and lattices as relations
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+  See commented-out notes for lattices as relations.
+
```

Comment:
Which ones ?

Reply:
Currently, there is no definition for `LatRel` in set.mm, and the related theorems are commented out. Maybe a separate issue should be created to revise and activate all these theorems in (current) subsection  9.2.6 "Posets and lattices as relations" (between ~tsrss and ~ledm). Or they should be deleted if they are useless. @digama0 you revised already some of these theorems in 2023 - what do you think? 

Nevertheless, I will merge this PR now (to see how the new structure will show up in HTML), because there are 4 approvals.

Reply:
> Currently, there is no definition for `LatRel` in set.mm, and the related theorems are commented out. Maybe a separate issue should be created to revise and activate all these theorems in (current) subsection 9.2.6 "Posets and lattices as relations" (between ~tsrss and ~ledm). Or they should be deleted if they are useless.

Issue #4302 opened for this topic.